test/RiskManagerV1.test.js

```diff
@@ -69,28 +73,181 @@ describe("RiskManagerV1", () => {
       })
     })
 
-    context("when deposit is in liquidation state", () => {
-      let notifyLiquidationTx
+    context(
+      "when deposit is in liquidation state and the surplus pool is empty",
```

Comment:
We could also have:
```
context("when deposit is in liquidation state", () => {
  it("should emit NotifiedLiquidation event", async () => {...}))
  context("when the surplus pool is empty, () => {...})
  context("when the surplus pool is smaller then the deposit lot size, () => {...})
  // (...)
})
```

Reply:
Improved the structure of the context tree in https://github.com/keep-network/coverage-pools/pull/43/commits/c151f7be59a000a30fc5eadee7ae89944d76fc5a. However, there is no possibility to check `NotifiedLiquidation` in one place as `beforeEach` hooks look slightly different in each case.